client/src/components/WeatherCard.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import './WeatherCard.css'
+
+function convertUnixTimestampToCityTime(unixTimestamp, utcOffsetInSeconds) {
+    const utcSeconds = unixTimestamp + utcOffsetInSeconds;
+    const cityDate = new Date(utcSeconds * 1000);
+
+    return cityDate.toLocaleTimeString(undefined, {
+        timeZone: 'UTC',
+        hour: 'numeric',
+        minute: 'numeric'
+    });
+}
+
+
+
+function WeatherCard({ data, city }) {
+     
+    if (data && city) {
+        return (
+            <div className='weatherCard'>
+                <h3>
+                    <span>{data.name}, {data.sys.country}</span>
+                </h3>
+                <h3>
+                    <span>{Math.round((data.main.temp-273.15) * 9/5 + 32)} Fahrenheit, {Math.round((data.main.temp-273.15))} Celsius </span>
+                </h3>
+                <h3>
+                    <span> Feels like: {Math.round((data.main.feels_like-273.15) * 9/5 + 32)} Fahrenheit, {Math.round((data.main.feels_like-273.15))} Celsius </span>
```

Comment:
Now that you're doing these conversions more than once, make a helper function to do them for you. E.g.

`Feels like {kelvinToFahrenheit(data.main.feels_like)} Fahrenheit, ...`

Notice how I named these helper functions. It's just just `to_fahrenheit`, because as I mentioned above, that documents the *output* but lacks units on the *input*.

Reply:
And @jiajdo try to avoid something we call "magic numbers". For example, 273.15, what is this? Generally speaking, it's best to define constant variables at the top of the code file (below the import lines) in all caps that help to describe what these hardcoded numbers are. Such as

import xxx
import xxx

const ABS_ZERO_IN_KELVIN_DEGREES = 273.15

Then use this highly descriptive constant in the code's calculations. 

We usually only hardcode -1, 0, and 1 without any constants defined for them. Any other number should be defined as a constant although occasionally 2 doesn't need it as long as the surrounding code context is super clear for anyone to know why 2 is hardcoded.

If there's going to be a hardcoded formula reused in more than one place, such as 9/5 + 32, this whole thing can be assigned to a constant as well.